keycloak4s-playground/src/test/resources/logback-test.xml

```diff
@@ -17,10 +17,10 @@
         </encoder>
     </appender>
 
-    <logger name="keycloak4s.admin" level="error"/>
-    <logger name="keycloak4s.auth"  level="error"/>
+    <logger name="keycloak4s.admin" level="trace"/>
```

Comment:
Should this still be set to trace? I see you changed it from error?

Reply:
Not sure why it wasn't on trace before, can't see a reason to restrict the tests' logs.